lib/csharp/thrift_netstd/Benchmarks/Thrift.Benchmarks/CompactProtocolBenchmarks.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the Apache Software Foundation(ASF) under one
```

Comment:
We should investigate whether this large pack of Thrift files is really needed as source in the plugin. It seems unncessary.